src/main/java/com/bgsoftware/superiorskyblock/core/menu/button/impl/menu/BiomeButton.java

```diff
@@ -83,6 +88,19 @@ public void onButtonClick(SuperiorSkyblockPlugin plugin, MenuBiomes superiorMenu
             return;
         }
 
+        File file = new File(plugin.getDataFolder(), "menus/biomes.yml");
```

Comment:
Every time someone clicks a button, you read a file from the disk.
This will kill servers due to bad performance.
Instead, when the menu is loaded alongside of buttons, you should read this value in that time and create the biomebutton with this value, cached.